packages/datepickers/src/styled/index.ts

```diff
@@ -5,5 +5,14 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-export * from './styled-menu';
-export * from './styled-datepicker';
+export { default as StyledMenu } from './StyledMenu';
+export { default as StyledDatepicker } from './StyledDatepicker';
+export { default as StyledRangeCalendar } from './StyledRangeCalendar';
+export { default as StyledHeader } from './StyledHeader';
+export { default as StyledHeaderPaddle } from './StyledHeaderPaddle';
+export { default as StyledHeaderLabel } from './StyledHeaderLabel';
+export { default as StyledCalendar } from './StyledCalendar';
+export { default as StyledCalendarItem } from './StyledCalendarItem';
+export { default as StyledDayLabel } from './StyledDayLabel';
+export { default as StyledHighlight } from './StyledHighlight';
+export { default as StyledDay } from './StyledDay';
```

Comment:
Default exports are a departure from what we've done with other CSS-in-JS refactors.

Reply:
Why was this resolved?

Reply:
I've corrected this to use named exports. Are you seeing the most up-to-date file?